.java-version

```diff
@@ -1 +1 @@
-oracle64-1.8.0.152
```

Comment:
This has existed in the repository for a while, and I've had no issues building. I no longer have this version of oracle's Java, which caused builds to fail (I use jenv). I've changed this to 1.8, but we may want to remove `.java-version` from the repo entirely if this causes problems.